app/templates/group.html

```diff
@@ -36,12 +36,14 @@
 						{%- endfor %}
 						</td>
 					</tr>
-					{%- if group.detail.bug_ticket %}
 					<tr>
 						<td>Ticket</td>
-						<td>{{ bug_ticket(group.detail.bug_ticket) }}</td>
+					        {%- if group.detail.bug_ticket %}
+                                                    <td>{{ bug_ticket(group.detail.bug_ticket) }}</td>
+                                                {%-else %}
```

Comment:
Please check the group status and only show the create link if the group is "Vulnerable".

It makes no sense to create a ticket for "fixed, testing or not affected". Additional its also pointless to create a ticket if the status is "Unknown".

Either kill the ticket row in such situations or show "None" instead.